pymedphys/_mosaiq/delivery.py

```diff
@@ -29,10 +29,9 @@
 
 import struct
 
-import attr
-
 import numpy as np
 
+import attr
```

Comment:
Any reason for changing this here?

Reply:
Humm.. I definetely not recall changing that. Would it be possible that poetry  or pylint add that line?

Reply:
If I run poetry run pre-commit run --all-files twice isort and black fail but the second time they pass. All the commits I've submitted have passed successfullly.
![2019-11-13_21-46](https://user-images.githubusercontent.com/8771255/68827487-6f77bf80-065f-11ea-9e49-009ee9188f2d.png)